video2dataset/subsamplers/cut_detection_subsampler.py

```diff
@@ -0,0 +1,31 @@
+from clipping_subsampler import ClippingSubsampler
+import numpy as np
+from scenedetect import detect, AdaptiveDetector
+
+
+class CutDetectionSubsampler:
+    def __init__(self, oom_clip_count, encode_formats, video_fps):
+        self.oom_clip_count = oom_clip_count
+        self.encode_formats = encode_formats
+        self.video_fps = video_fps
+
+        self.clip_subsampler = ClippingSubsampler(oom_clip_count, encode_formats)
```

Comment:
subsamplers shouldn't use other subsamplers, you want to implement this in worker.py